src/pe_source/cybersixgill.py

```diff
@@ -0,0 +1,298 @@
+"""Collect Cybersixgill data."""
+
+# Standard Python Libraries
+from datetime import date, datetime, timedelta
+import logging
+import sys
+
+from .data.pe_db.db_query import (
+    get_breaches,
+    get_data_source_uid,
+    get_orgs,
+    insert_sixgill_alerts,
+    insert_sixgill_breaches,
+    insert_sixgill_credentials,
+    insert_sixgill_mentions,
+    insert_sixgill_topCVEs,
+)
+from .data.sixgill.api import get_sixgill_organizations
+from .data.sixgill.source import (
+    alerts,
+    alias_organization,
+    creds,
+    cve_summary,
+    mentions,
+    root_domains,
+    top_cves,
+)
+
+# Set todays date formatted YYYY-MM-DD and the start_date 30 days prior
+TODAY = date.today()
+DAYS_BACK = timedelta(days=30)
+START_DATE = str(TODAY - DAYS_BACK)
+END_DATE = str(TODAY)
+DATE_SPAN = f"[{START_DATE} TO {END_DATE}]"
+
+# Set dates to YYY-MM-DD H:M:S format
+NOW = datetime.now()
+BACK = timedelta(days=30)
+FROM_DATE = (NOW - BACK).strftime("%Y-%m-%d %H:%M:%S")
+TO_DATE = NOW.strftime("%Y-%m-%d %H:%M:%S")
+
+
+class Cybersixgill:
+    """Fetch Cybersixgill data."""
+
+    def __init__(self, orgs_list, method_list):
+        """Initialize Cybersixgill class."""
+        self.orgs_list = orgs_list
+        self.method_list = method_list
+
+    def run_cybersixgill(self):
+        """Run Cybersixgill api calls."""
+        orgs_list = self.orgs_list
+        method_list = self.method_list
+
+        # Get org info from PE database
+        pe_orgs = get_orgs()
+
+        # Get Cybersixgill org info
+        sixgill_orgs = get_sixgill_organizations()
+        failed = []
+        count = 0
+
+        # Get data source uid
+        source_uid = get_data_source_uid("Cybersixgill")
+
+        # Run top CVEs. Same for all orgs
+        if "topCVEs" in method_list:
+            topCVEs = self.get_topCVEs(source_uid)
+            if topCVEs == 1:
```

Comment:
Remove unneeded variable:
```suggestion
            if self.get_topCVEs(source_uid) == 1:
```